packages/apps/graph-client/src/constants/search.ts

```diff
@@ -0,0 +1,38 @@
+export enum SearchType {
+  GasEstimation = 'gasEstimation',
+  Account = 'account',
+  Block = 'block',
+  Transactions = 'request-key',
+  Event = 'event',
+}
+
+export const searchTypeLabels: Record<string, string> = {
+  'request-key': 'Request Key',
+  account: 'Account',
+  event: 'Event Name',
+  block: 'Block Hash',
+  gasEstimation: 'Cmd',
+};
+export const secondSearchTypeLabels: Record<string, string> = {
+  account: 'Module',
+  gasEstimation: 'Hash',
+};
+export const thirdSeachTypeLabels: Record<string, string> = {
+  gasEstimation: 'Signatures',
+};
+export const searchTypePlaceholders: Record<string, string> = {
+  'request-key': 'vCiATVJgm7...',
+  account: 'k:1234...',
+  event: 'coin.TRANSFER',
+  block: 'CA9orP2yM...',
+  gasEstimation: 'cmd',
```

Comment:
This could be something like `(coin.details k:1234...` right?

Reply:
Fixed